### ISSUE TITLE
index_years = 1 or NA

### ISSUE THREAD

**Issue by arni-magnusson** [DanOvando/sraplus#41]
When looking through the default values in format_driors(), this one raises an eyebrow or two:

https://github.com/DanOvando/sraplus/blob/ac05e92cf5ea43b914bb5e0aac7648fb54e8f9c3/R/format-driors.R#L66

Firstly, similar arguments such as `effort_years` have a default value of `NA`.

Secondly, meaningful values for these arguments would be more recent years, such as 1990 or 2020. The year 1 AD seems like an odd default value.

Would it make sense to have the default value of `index_years` as `NA`? This is not super important, but becomes somewhat relevant when writing functions that call format_driors, handling the large number of diverse arguments.

**Comment by DanOvando** [comment]
Huh that is odd, don't remember why I did it that way. I'll give it a quick check to make sure there isn't some reason for that but I suspect you are correct that would be better as an NA

**Comment by DanOvando** [comment]
OK, it's an artifact. The model itself tracks years relative to the first year of the catches internally, and so 1 means "first year of the catch data" not "1 AD" ha. It didn't affect anything, but agreed it is confusing. Replaced the default with NA and it seems some other check I added at some point fixed whatever reason I had for that being the default, so pushing the new version with `index_years = NA` as default. Thanks!
